bibliopixel/remote/control.py

```diff
@@ -34,10 +34,20 @@ def __init__(self, layout, animations,
                  external_access=False, port=5000,
                  title='BiblioPixel Remote', bgcolor='black',
                  font_color='white', default=None,
-                 triggers=[]):
+                 triggers=[], auto_demo=None):
 
         self.internal_delay = 0  # never wait
 
+        if auto_demo:
```

Comment:
auto_demo would look something like this:
```
"auto_demo": {
            "name": "Demo Mode",
            "random": true,
            "run": {
                "seconds": 30
            },
            "data": {
                "bgcolor": "blue"
            }
        }
```

What this does is loops through all given animations and builds a single Demo sequence out of every other animation given to `remote` and pop `run` off this config and apply it to each sub-animation. `run` is not needed for the demo sequence as it's a sequence and is unbounded. Thought about naming it something else, but it applies here. The rest of the values are standard `sequence` or `remote` animation params.

Basically, this saves me hours of making `sequence` entries that encompass all the other animations I already setup. This way you can either hit demo, or hit each on their own.